### DISCUSSION TITLE
Does this work for zone sync server

### DISCUSSION THREAD

**Discussion by aksonkar** [nicholaschiasson/ngx_upstream_jdomain#83] [Q&A]
I am currently working on zone sync for key val storage. I would like to use the address of my cluster which is the address of google ilb sitting in front of my nginx nodes. Does this module work for zone sync server directive?

**Comment by nicholaschiasson** [comment]
Hi @aksonkar, sorry for the late response. If I understand correctly, you have an nginx commercial subscription, correct? I am assuming this since the zone sync module is only available as part of the commercial subscription. In that case, aside from the fact that I don't think this module helps in that use case, this module becomes unnecessary. The zone sync server directive has an optional `resolve` argument which accomplishes what this module does, and in a way that is supported directly by the Nginx team. I would suggest using that.

If I didn't answer your question properly, please feel free to reopen this 😄
